<!DOCTYPE html>
<html>
  <head>
    
    <meta name="google-site-verification" content="NGVGrsKdgzybUNks-ZTxt3-Z90KYQAj1_a_OhwOCTyI" />
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width initial-scale=1" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge">

    <title>Projects</title>
    <meta name="description" content="The Morin Laboratory performs cutting-edge lymphoma research using a combination of dry lab and wet lab experiments.
">

    <link rel="stylesheet" href="/css/main.css">
    <link rel="canonical" href="http://localhost:4000/projects/">

    <script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.4/jquery.min.js"></script>


    <!-- Begin Jekyll SEO tag v2.8.0 -->
<title>Projects | Translational lymphoma research in the Morin laboratory at Simon Fraser University</title>
<meta name="generator" content="Jekyll v3.9.3" />
<meta property="og:title" content="Projects" />
<meta name="author" content="Bruno Grande, Ryan Morin and the Morin laboratory
" />
<meta property="og:locale" content="en_US" />
<meta name="description" content="The Morin Laboratory performs cutting-edge lymphoma research using a combination of dry lab and wet lab experiments." />
<meta property="og:description" content="The Morin Laboratory performs cutting-edge lymphoma research using a combination of dry lab and wet lab experiments." />
<link rel="canonical" href="http://localhost:4000/projects/" />
<meta property="og:url" content="http://localhost:4000/projects/" />
<meta property="og:site_name" content="Translational lymphoma research in the Morin laboratory at Simon Fraser University" />
<meta property="og:type" content="website" />
<meta name="twitter:card" content="summary" />
<meta property="twitter:title" content="Projects" />
<script type="application/ld+json">
{"@context":"https://schema.org","@type":"WebPage","author":{"@type":"Person","name":"Bruno Grande, Ryan Morin and the Morin laboratory\n"},"description":"The Morin Laboratory performs cutting-edge lymphoma research using a combination of dry lab and wet lab experiments.","headline":"Projects","url":"http://localhost:4000/projects/"}</script>
<!-- End Jekyll SEO tag -->

  </head>
  <body>

    <header class="site-header">

  <div class="wrapper">

    <a class="site-title" href="/">
      Morin Laboratory
      
        <span class="title-institution">  @  SFU</span>
      
    </a>

  </div>

  <div class="wrapper main-menu">

    <nav class="site-nav">
      <a href="#" class="menu-icon">
        <svg viewBox="0 0 18 15">
          <path fill="#424242" d="M18,1.484c0,0.82-0.665,1.484-1.484,1.484H1.484C0.665,2.969,0,2.304,0,1.484l0,0C0,0.665,0.665,0,1.484,0 h15.031C17.335,0,18,0.665,18,1.484L18,1.484z"/>
          <path fill="#424242" d="M18,7.516C18,8.335,17.335,9,16.516,9H1.484C0.665,9,0,8.335,0,7.516l0,0c0-0.82,0.665-1.484,1.484-1.484 h15.031C17.335,6.031,18,6.696,18,7.516L18,7.516z"/>
          <path fill="#424242" d="M18,13.516C18,14.335,17.335,15,16.516,15H1.484C0.665,15,0,14.335,0,13.516l0,0 c0-0.82,0.665-1.484,1.484-1.484h15.031C17.335,12.031,18,12.696,18,13.516L18,13.516z"/>
        </svg>
      </a>

      <div class="trigger">
        <ul>
          
          
            
          
            
          
            
          
            
          
            
          
            
          
            
            <li><a class="page-link" href="/projects/">
                Projects
            </a></li>
            
          
            
            <li><a class="page-link" href="/team/">
                Team
            </a></li>
            
          
            
            <li><a class="page-link" href="/software/">
                Software
            </a></li>
            
          
            
            <li><a class="page-link" href="/publications/">
                Publications
            </a></li>
            
          
            
          
            
            <li><a class="page-link" href="/pi/">
                GAMBL
            </a></li>
            
          
            
            <li><a class="page-link" href="/contact/">
                Contact
            </a></li>
            
          
        </ul>
        <div style="clear:both;"></div>
      </div>
    </nav>

  </div>

</header>


    <div class="page-content">
      <div class="wrapper">
        <div class="post">

  <header class="post-header">
    <h1 class="post-title">Projects</h1>
  </header>

  <article class="post-content">
    <div class="lab-wrapper">
    <ul class="lab-list">
    
    
        <li>
            <h2>Genetic and molecular mechanisms of treatment resistance in DLBCL</h2>
            
                <img class="float-right projects-photo" src="/images/projects/dhitsig.png" />
            
            
                <p><b>Funding: </b>Terry Fox Research Institute</p>
            
            
                <p><b>Collaborators: </b>Christian Steidl, David Scott, Nathalie Johnson</p>
            
            
                <p><b>Assignees: </b>Chris Rushton, Laura Hilton, Jeffrey Tang, Miguel Alcaide, Aixiang Jiang</p>
            
            <p>Non-Hodgkin lymphomas (NHL) represent the sixth most commonly diagnosed type of cancer in Canada with diffuse large B-cell lymphoma (DLBCL) being the most prevalent. DLBCL patients exhibit variable responses to available therapies such as R-CHOP. Some biomarkers beginning to enter clinical trials rely on gene expression signatures to identify high-risk cases. For example, DLBCL comprises two cell-of-origin (COO) subgroups with activated B-cell (ABC) cases typically having worse prognosis than germinal centre B-cell (GCB) cases. Therapeutics that exploit molecular features unique to each subgroup are being actively pursued but up-front identification of patients unlikely to achieve a durable response remains a challenge. Although existing methods can identify specific prognostic gene expression features, these capture only some of the clinical heterogeneity and require tumour RNA, which is not always available. Using comprehensive genomic analyses and by integrating mutation and expression data from over 2000 DLBCL tumours, we seek novel molecular features that are associated with treatment resistance in DLBCL. The ultimate goal of this work is to develop new predictive and prognostic biomarkers and identify new therapeutic targets for treatment-resistant DLBCLs.
</p>
        </li>
    
    
    
        <li>
            <h2>The role of the Fc Gamma receptor locus in resistance to R-CHOP</h2>
            
                <img class="float-right projects-photo" src="/images/projects/FCGR2B_cropped.png" />
            
            
                <p><b>Funding: </b>ASH Foundation</p>
            
            
                <p><b>Collaborators: </b>David Scott</p>
            
            
                <p><b>Assignees: </b>Laura Hilton, Miguel Alcaide, Jack Hillman</p>
            
            <p>Through genomic characterization of DLBCL we identified somatic amplification of the Fc gamma receptor 2B gene (FCGR2B) as a feature in ~5% of cases. We also  found the expression of FCGR2B (by RNA-seq) was prognostic in multiple DLBCL cohorts.  FCGR2B contributes to treatment resistance by promoting the internalization of the  rituximab antibody by malignant cells, allowing them to evade immune detection.  It may therefore serve as an important biomarker to identify high-risk patients and  stratify them to recieve newer immunotherapies that can mitigate the effects of high  FCGR2B expression. Understanding the genetic mechanisms that drive FCGR2B amplification  and overexpression has been complicated by the segmental duplication that affects  the FCGR locus, which makes it difficult to accurately align short reads to FCGR2B  or its closely-related paralog FCGR2C. We have been employing long-read Oxford  Nanopore sequencing and Bionano optical mapping to accurately characterize the  structural and single nucleotide variation within the locus.
</p>
        </li>
    
    
    
        <li>
            <h2>Resolving clonal structure and evolution patterns in DLBCL</h2>
            
                <img class="float-right projects-photo" src="/images/projects/DLBCL_Evolution.png" />
            
            
                <p><b>Funding: </b>Terry Fox Research Institute</p>
            
            
                <p><b>Collaborators: </b>Christian Steidl, David Scott, Nathalie Johnson</p>
            
            
                <p><b>Assignees: </b>Chris Rushton, Miguel Alcaide, Sarah Arthur</p>
            
            <p>Diffuse Large B-Cell Lymphoma is a genetically heterogeneous form of cancer,  with different cells in the tumour acquiring different mutations. This  intra-tumour heterogeneity can have profound effects on treatment; if a clonal  subpopulations is resistant to treatment, it will reform the tumour following  treatment, causing the patient to relapse. We are recieving samples from several clinical trials exploring different salvage therapies for patients with relapsed DLBCL. Tumour biopsies and liquid biopsies are being collected from patients  both before and after they are treated. Using these temporal sources of tumour DNA, we can identify clonal subpopulations in the tumour,  and further explore how the tumour evolves and adapts in the face of a strong  selective pressure such as treatment. The mutations unique to these resistant  clonal subpopulations can be explored to determine which mutation(s) result  in treatment resistance. We can further determine if these mutations are  detectable prior to treatment, which may allow patient to be screened for resistance  mutations before they are treated.
</p>
        </li>
    
    
    
        <li>
            <h2>Circulating Tumour DNA as a biomarker in non-Hodgkin lymphoma</h2>
            
                <img class="float-right projects-photo" src="/images/projects/Figure_1_amplicon.png" />
            
            
                <p><b>Funding: </b>CIHR</p>
            
            
                <p><b>Collaborators: </b>Christian Steidl, Nathalie Johnson</p>
            
            
                <p><b>Assignees: </b>Miguel Alcaide, Stephen Yu, Chris Rushton</p>
            
            <p>This project aims to develop a novel way of monitoring patients with lymphoma and find a new immunotherapy regimen that would be effective in patients that are not cured with conventional chemotherapy. They will first genetically profile the tumours to determine at which time point they become resistant. They will then determine how these lymphomas evade immune detection and test if targeting these evasion strategies, i.e. "harnessing the immune response" is successful at eradicating chemo-resistant lymphoma.
</p>
        </li>
    
    
    
        <li>
            <h2>Comprehensive proteogenomic characterization of mantle cell lymphoma (MCL)</h2>
            
                <img class="float-right projects-photo" src="/images/projects/mcl_oncoplot.png" />
            
            
                <p><b>Funding: </b>Terry Fox Research Institute</p>
            
            
                <p><b>Collaborators: </b>Marco Marra, David Scott, Diego Villa</p>
            
            
                <p><b>Assignees: </b>Prasath Pararajalingam, Krysta Coyle</p>
            
            <p>Mantle cell lymphoma is a rare cancer that derives from B lymphocytes in the periphery (mantle zone) of the germinal centre. MCL is most often diagnosed at stage 3 or 4 and is typically aggressive although some patients have an indolent form of the disesae. Only 30-40% of patients with advanced disease achieving a complete response to frontline treatments (R-bendamustine). Heterogeneous clinical response is partially attributable to observable variations in morphology, immunophenotype and proliferative capacity; however, there is little known about the relationship between genetic features and variation in phenotype or response to specific treatment regimens. Through a meta-analysis of in-house and existing exome sequencing data, our group has identified multiple novel driver mutations in MCL including some that associate with high or low proliferation. To more completely ascertain the mutations that lead to lymphomagenesis in MCL, we are performing an integrative proteogenomic characterization of MCL tumours using a combination of whole genome sequencing, RNA-seq and shotgun proteomics. We will are using these data to identify novel coding and non-coding (i.e. regulatory) driver mutations that affect the natural history of MCL including treatment response.
</p>
        </li>
    
    
    
        <li>
            <h2>Identification and functional characterization of recurrent non-coding mutations in NHLs</h2>
            
                <img class="float-right projects-photo" src="/images/projects/NFKBIZ_overview_cropped.png" />
            
            
                <p><b>Funding: </b>Terry Fox Research Institute, CIHR</p>
            
            
                <p><b>Collaborators: </b>Christian Steidl, Tim Audas, Peter Unrau</p>
            
            
                <p><b>Assignees: </b>Sarah Arthur, Krysta Coyle, Matthew Nguyen, Aixiang Jiang</p>
            
            <p>The list of protein-coding drivers identified through sequencing in DLBCL has begun to saturate and yet there remains a gap in our understanding of how these genetic alterations contribute to the transcriptional differences underlying known prognostic signatures and molecular subgroups. For example, dysregulation of NF-κB is the hallmark feature of ABC DLBCL but many ABC cases appear to lack the common driver mutations that are known to promote NF-κB. We are seeking driver mutations that contribute to some features of NHLs. We use data-driven comprehnsive methods that allow identification of non-coding SSMs with a regulatory effect in cis or trans on genes/proteins relevant to malignancy.
</p>
        </li>
    
    
    
        <li>
            <h2>Comprehensive characterization of endemic and sporadic Burkitt lymphoma</h2>
            
                <img class="float-right projects-photo" src="/images/projects/blgsp_ash_2017.jpg" />
            
            
                <p><b>Funding: </b>National Cancer Institute, Foundation for Burkitt Lymphoma Research</p>
            
            
                <p><b>Collaborators: </b>Marco Marra, Louis Staudt</p>
            
            
                <p><b>Assignees: </b>Bruno Grande, Aixiang Jiang</p>
            
            <p>Burkitt lymphoma (BL) is currently classified as one of three clinical variants: endemic BL, sporadic BL, and HIV-associated BL. The endemic variant is primarily diagnosed in sub-Saharan Africa and is epidemiologically associated with the Epstein–Barr virus (EBV) and malaria infection; although, the role of these pathogens in promoting the formation of BL remains unclear. On the other hand, sporadic BL is diagnosed outside of malaria-endemic areas, with only a minority of cases being EBV-positive (10-30%). This project seeks to generate high-throughput whole genome and transcriptome sequencing data from over 100 BL tumours in order to elucidate the genetic and molecular underpinnings of the disease. With this project, we aim to refine the mutational landscape of BL to facilitate the development of less toxic targeted therapies. We also plan on investigating the oncogenic role of EBV and whether it may serve as a better criterion for disease classification than the current system relying on geographic origin.
</p>
        </li>
    
    
    
        <li>
            <h2>Genetic characterization of HIV-associated DLBCL</h2>
            
            
                <p><b>Funding: </b>National Cancer Institute</p>
            
            
                <p><b>Collaborators: </b>Marco Marra, Louis Staudt</p>
            
            
                <p><b>Assignees: </b>Nicole Thomas, Bruno Grande</p>
            
            <p>Diffuse large B-cell lymphoma is a common HIV-associated cancer. The genetic and molecular aetiology of HIV+ DLBCL has not been thoroughly explored.  This study seeks to characterize HIV+ DLBCL through comprehensive whole genome,  transcriptome, and miRNA sequencing. Enhancing our understanding of the molecular features of these  tumours, particularly those that distinguish them from other DLBCLs, may translate into improved therapies for a growing population of patients afflicted with both HIV and cancer. 
</p>
        </li>
    
    
    
        <li>
            <h2>Genetic characterization of canine B-cell lymphomas</h2>
            
                <img class="float-right projects-photo" src="/images/projects/TRAF3.png" />
            
            
                <p><b>Funding: </b>NSERC</p>
            
            
            
                <p><b>Assignees: </b>Kevin Bushell, Jack Hillman, Matt Cheung</p>
            
            <p>Cancer is the leading cause of premature death in pet dogs and non-Hodgkin lymphomas are the most common cancer in most dog breeds. Unlike human lymphomas with similar histopathology, canine B-cell lymphomas (cBCL) are incurable and exhibit an agressive clinical course. We are sequencing cBCLs to identify driver mutations that contribute to this cancer in a variety of dog breeds.  We identified TRAF3 as a novel tumour suppressor in cBCL and are pursuing additional candidate genes.  We are also exploring plasma from cBCLs to evaluate the utility of ctDNA as a biomarker and non-invasive source of tumour DNA in canine lymphoma. 
</p>
        </li>
    
    
    </ul>
</div>

  </article>

</div>

      </div>
    </div>

    <footer class="site-footer">

  <div class="wrapper">

    <div class="footer-col-wrapper">
      <div class="footer-col footer-col-1">
        <p>The Morin Laboratory performs cutting-edge lymphoma research using a combination of dry lab and wet lab experiments.
</p>
      </div>

      <div class="footer-col footer-col-2">
        <ul class="social-media-list">
          
          <li>
            <a href="https://twitter.com/morinryan"><span class="icon icon--twitter"><svg viewBox="0 0 16 16"><path fill="#828282" d="M15.969,3.058c-0.586,0.26-1.217,0.436-1.878,0.515c0.675-0.405,1.194-1.045,1.438-1.809c-0.632,0.375-1.332,0.647-2.076,0.793c-0.596-0.636-1.446-1.033-2.387-1.033c-1.806,0-3.27,1.464-3.27,3.27 c0,0.256,0.029,0.506,0.085,0.745C5.163,5.404,2.753,4.102,1.14,2.124C0.859,2.607,0.698,3.168,0.698,3.767 c0,1.134,0.577,2.135,1.455,2.722C1.616,6.472,1.112,6.325,0.671,6.08c0,0.014,0,0.027,0,0.041c0,1.584,1.127,2.906,2.623,3.206 C3.02,9.402,2.731,9.442,2.433,9.442c-0.211,0-0.416-0.021-0.615-0.059c0.416,1.299,1.624,2.245,3.055,2.271 c-1.119,0.877-2.529,1.4-4.061,1.4c-0.264,0-0.524-0.015-0.78-0.046c1.447,0.928,3.166,1.469,5.013,1.469 c6.015,0,9.304-4.983,9.304-9.304c0-0.142-0.003-0.283-0.009-0.423C14.976,4.29,15.531,3.714,15.969,3.058z"/></svg>
</span><span class="username">morinryan</span></a>

          </li>
          

          
          <li>
            <a href="https://github.com/morinlab"><span class="icon icon--github"><svg viewBox="0 0 16 16"><path fill="#828282" d="M7.999,0.431c-4.285,0-7.76,3.474-7.76,7.761 c0,3.428,2.223,6.337,5.307,7.363c0.388,0.071,0.53-0.168,0.53-0.374c0-0.184-0.007-0.672-0.01-1.32 c-2.159,0.469-2.614-1.04-2.614-1.04c-0.353-0.896-0.862-1.135-0.862-1.135c-0.705-0.481,0.053-0.472,0.053-0.472 c0.779,0.055,1.189,0.8,1.189,0.8c0.692,1.186,1.816,0.843,2.258,0.645c0.071-0.502,0.271-0.843,0.493-1.037 C4.86,11.425,3.049,10.76,3.049,7.786c0-0.847,0.302-1.54,0.799-2.082C3.768,5.507,3.501,4.718,3.924,3.65 c0,0,0.652-0.209,2.134,0.796C6.677,4.273,7.34,4.187,8,4.184c0.659,0.003,1.323,0.089,1.943,0.261 c1.482-1.004,2.132-0.796,2.132-0.796c0.423,1.068,0.157,1.857,0.077,2.054c0.497,0.542,0.798,1.235,0.798,2.082 c0,2.981-1.814,3.637-3.543,3.829c0.279,0.24,0.527,0.713,0.527,1.437c0,1.037-0.01,1.874-0.01,2.129 c0,0.208,0.14,0.449,0.534,0.373c3.081-1.028,5.302-3.935,5.302-7.362C15.76,3.906,12.285,0.431,7.999,0.431z"/></svg>
</span><span class="username">morinlab</span></a>

          </li>
          
        </ul>
      </div>
    </div>

  </div>

</footer>


  </body>

</html>
