<!DOCTYPE html>
<html>
  <head>
    
    <meta name="google-site-verification" content="NGVGrsKdgzybUNks-ZTxt3-Z90KYQAj1_a_OhwOCTyI" />
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width initial-scale=1" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge">

    <title>Team</title>
    <meta name="description" content="The Morin Laboratory performs cutting-edge lymphoma research using a combination of dry lab and wet lab experiments.
">

    <link rel="stylesheet" href="/css/main.css">
    <link rel="canonical" href="http://localhost:4000/team/">

    <script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.4/jquery.min.js"></script>


    <!-- Begin Jekyll SEO tag v2.8.0 -->
<title>Team | Translational lymphoma research in the Morin laboratory at Simon Fraser University</title>
<meta name="generator" content="Jekyll v3.9.3" />
<meta property="og:title" content="Team" />
<meta name="author" content="Bruno Grande, Ryan Morin and the Morin laboratory
" />
<meta property="og:locale" content="en_US" />
<meta name="description" content="The Morin Laboratory performs cutting-edge lymphoma research using a combination of dry lab and wet lab experiments." />
<meta property="og:description" content="The Morin Laboratory performs cutting-edge lymphoma research using a combination of dry lab and wet lab experiments." />
<link rel="canonical" href="http://localhost:4000/team/" />
<meta property="og:url" content="http://localhost:4000/team/" />
<meta property="og:site_name" content="Translational lymphoma research in the Morin laboratory at Simon Fraser University" />
<meta property="og:type" content="website" />
<meta name="twitter:card" content="summary" />
<meta property="twitter:title" content="Team" />
<script type="application/ld+json">
{"@context":"https://schema.org","@type":"WebPage","author":{"@type":"Person","name":"Bruno Grande, Ryan Morin and the Morin laboratory\n"},"description":"The Morin Laboratory performs cutting-edge lymphoma research using a combination of dry lab and wet lab experiments.","headline":"Team","url":"http://localhost:4000/team/"}</script>
<!-- End Jekyll SEO tag -->

  </head>
  <body>

    <header class="site-header">

  <div class="wrapper">

    <a class="site-title" href="/">
      Morin Laboratory
      
        <span class="title-institution">  @  SFU</span>
      
    </a>

  </div>

  <div class="wrapper main-menu">

    <nav class="site-nav">
      <a href="#" class="menu-icon">
        <svg viewBox="0 0 18 15">
          <path fill="#424242" d="M18,1.484c0,0.82-0.665,1.484-1.484,1.484H1.484C0.665,2.969,0,2.304,0,1.484l0,0C0,0.665,0.665,0,1.484,0 h15.031C17.335,0,18,0.665,18,1.484L18,1.484z"/>
          <path fill="#424242" d="M18,7.516C18,8.335,17.335,9,16.516,9H1.484C0.665,9,0,8.335,0,7.516l0,0c0-0.82,0.665-1.484,1.484-1.484 h15.031C17.335,6.031,18,6.696,18,7.516L18,7.516z"/>
          <path fill="#424242" d="M18,13.516C18,14.335,17.335,15,16.516,15H1.484C0.665,15,0,14.335,0,13.516l0,0 c0-0.82,0.665-1.484,1.484-1.484h15.031C17.335,12.031,18,12.696,18,13.516L18,13.516z"/>
        </svg>
      </a>

      <div class="trigger">
        <ul>
          
          
            
          
            
          
            
          
            
          
            
          
            
          
            
            <li><a class="page-link" href="/projects/">
                Projects
            </a></li>
            
          
            
            <li><a class="page-link" href="/team/">
                Team
            </a></li>
            
          
            
            <li><a class="page-link" href="/software/">
                Software
            </a></li>
            
          
            
            <li><a class="page-link" href="/publications/">
                Publications
            </a></li>
            
          
            
          
            
            <li><a class="page-link" href="/pi/">
                GAMBL
            </a></li>
            
          
            
            <li><a class="page-link" href="/contact/">
                Contact
            </a></li>
            
          
        </ul>
        <div style="clear:both;"></div>
      </div>
    </nav>

  </div>

</header>


    <div class="page-content">
      <div class="wrapper">
        <div class="post">

  <header class="post-header">
    <h1 class="post-title">Team</h1>
  </header>

  <article class="post-content">
    <div style="margin-bottom: 2em;">
    <img src="/images/team/morinlab_pool_2022_crop.png" />
</div>

<div class="lab-wrapper">
    <ul class="lab-list">
    <!-- Current PIs -->
    
        
            
                <li>
    <h2><span class="member-name">Dr. Ryan D. Morin</span>, Associate Professor</h2>
    
        <img class="float-left member-photo" src="/images/team/Morin_headshot_2019.gif" />
    
    <p>I am in the department of <a href="http://www.sfu.ca/mbb.html">Molecular Biology and Biochemistry</a> at <a href="http://www.sfu.ca">Simon Fraser University</a> and hold a Scientist appointment in the <a href="http://www.bcgsc.ca">BC Cancer Genome Sciences Centre</a>. I completed my MSc and PhD in bioinformatics at UBC in the laboratory of Marco Marra. My early research involved the use of massively parallel sequencing to identify driver mutations in leukemias and lymphomas including histone modifiers such as <a href="https://www.ncbi.nlm.nih.gov/pubmed/20081860">EZH2</a> and <a href="https://www.ncbi.nlm.nih.gov/pubmed/21796119">KMT2D</a>. My current research focus involves using genome and transcriptome sequencing to discover novel drivers in B-cell lymphomas and to determine mechanisms that underlie tumour progression and treatment resistance. I have a general interest in developing methods to identify non-coding mutations with regulatory potential from genome-wide data sets and error-corrected sequencing strategies and algorithms to identify mutations and copy number from circulating tumour DNA (<a href="https://www.ncbi.nlm.nih.gov/pubmed/28874686">ctDNA</a>). Dr. Morin is a Scholar of the <a href="https://www.msfhr.org/">Michael Smith Foundation for Health Research</a>.
</p>
    
        <a href="https://orcid.org/0000-0003-2932-7800"><img src="/images/team/orcid.png" width="50" height="50" /></a>
    
    
    <a href="https://scholar.google.ca/citations?user=IDau0mkAAAAJ&amp;hl=en"><img src="/images/team/scholar.png" width="60" height="60" /></a>
    
    
    <a href="https://www.researchgate.net/profile/Ryan_Morin"><img src="/images/team/researchgate.png" width="60" height="60" /></a>
    
    
    <a href="https://github.com/rdmorin"><img src="/images/team/github.jpg" width="52" height="52" /></a>
    
    
    <a href="https://twitter.com/morinryan"><img src="/images/team/X.png" width="55" height="55" /></a>
    
</li>

            
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
    <!-- Current non-PIs -->
    
        
    
        
            
                <li>
    <h2><span class="member-name">Dr. Laura Hilton</span>, Staff Scientist, BC Cancer Centre for Lymphoid Cancer</h2>
    
        <img class="float-left member-photo" src="/images/team/lhilton.jpg" />
    
    <p>B-cell non-Hodgkin lymphomas have a high frequency of driver structural variants, especially those that involve placing the oncogenes MYC, BCL2, BCL6, and CCND1 under the control of potent immunoglobulin enhancers. My project utilizes whole genome and targeted sequencing to characterize these driver SVs across B-NHLs with a goal of understanding the mechanisms by which they arise. I'm also interested in using gene expression and mutation data to facilitate the classification of rare and/or poorly characterized lymphomas, to identify progonostic and predictive biomarkers and targetable vulnerabilities that can be exploited in the development of novel therapies.
</p>
    
        <a href="https://orcid.org/0000-0002-6413-6586"><img src="/images/team/orcid.png" width="50" height="50" /></a>
    
    
    
    
    <a href="https://github.com/lkhilton"><img src="/images/team/github.jpg" width="52" height="52" /></a>
    
    
</li>

            
        
    
        
            
                <li>
    <h2><span class="member-name">Dr. Krysta Coyle</span>, Postdoctoral Fellow</h2>
    
        <img class="float-left member-photo" src="/images/team/kcoyle.jpg" />
    
    <p>My research aims to understand the role of recurrent cis-regulatory mutations in mantle cell lymphoma (MCL). Specifically, I am characterizing a potential role for alternative splicing in the progression of MCL. This work uses a variety of molecular biology tools and bioinformatic approaches to determine how mutations in RNA binding proteins affect in vitro cell behaviour. I am a passionate advocate for equity, diversity, and inclusion and enjoy communicating science to kids and adults alike.
</p>
    
        <a href="http://orcid.org/0000-0002-1309-4873"><img src="/images/team/orcid.png" width="50" height="50" /></a>
    
    
    <a href="https://scholar.google.ca/citations?user=AeVci4cAAAAJ&amp;hl=en"><img src="/images/team/scholar.png" width="60" height="60" /></a>
    
    
    <a href="https://www.researchgate.net/profile/Krysta_Coyle"><img src="/images/team/researchgate.png" width="60" height="60" /></a>
    
    
    <a href="https://github.com/kmcoyle"><img src="/images/team/github.jpg" width="52" height="52" /></a>
    
    
    <a href="https://twitter.com/microbialkrysta"><img src="/images/team/twitter.png" width="55" height="55" /></a>
    
</li>

            
        
    
        
            
                <li>
    <h2><span class="member-name">Prasath Pararajalingam</span>, PhD Student</h2>
    
        <img class="float-left member-photo" src="/images/team/ppararajalingam.jpg" />
    
    <p>My PhD project has me exploring the molecular aetiology of mantle cell lymphoma (MCL) using a combination of data types and a variety of bioinformatic techniques. My analysis of the mutational landscape of MCL has revealed an enrichment of mutations in splicing factors and RNA-binding proteins. Currently, I am investigating deregulation of non-coding elements using a large dataset of WGS and RNA-seq MCL tumours.
</p>
    
    
    
    
    <a href="https://github.com/ppararaj"><img src="/images/team/github.jpg" width="52" height="52" /></a>
    
    
</li>

            
        
    
        
            
                <li>
    <h2><span class="member-name">Haya Shaalan</span>, MSc Student</h2>
    
        <img class="float-left member-photo" src="/images/team/haya.jpg" />
    
    <p>I work on implementing and improving pipelines for our Genomic Analysis of Mature B-cell Lymphoma project. I’ve taken an interest in incorporating promethION long-read sequencing data into this project to perform various analyses exploring copy number variation, simple somatic mutations, and structural variation patterns. PromethION sequencing has certain advantages over short read sequencing in that it provides a more comprehensive view on structural variant discovery and informs on methylation state. We hope that the addition of promethION data will consolidate previous findings, as well as provide further insight to the genomic landscape of this disease.
</p>
    
    
    
    
    <a href="https://github.com/hayashaalan"><img src="/images/team/github.jpg" width="52" height="52" /></a>
    
    
</li>

            
        
    
        
            
                <li>
    <h2><span class="member-name">Nicole Thomas</span>, PhD Student</h2>
    
        <img class="float-left member-photo" src="/images/team/nthomas.jpg" />
    
    <p>I completed my Honours Project in the Morin lab, during which I investigated the role of putative enhancers in Mantle Cell Lymphoma. My current role in the lab involves the mining of RNA-Seq data to identify unique gene expression patterns present in Non-Hodgkin’s Lymphoma. Currently, I am using RNA-Seq data, in conjunction with Whole Genome Sequencing data, to identify mutations and single nucleotide polymorphisms that affect the alternative splicing of RNA transcripts.
</p>
    
    
    
    
    <a href="https://github.com/nthomas11"><img src="/images/team/github.jpg" width="52" height="52" /></a>
    
    
</li>

            
        
    
        
            
                <li>
    <h2><span class="member-name">Dr. Kostiantyn Dreval</span>, Research Associate</h2>
    
        <img class="float-left member-photo" src="/images/team/kdreval.jpg" />
    
    <p>I obtained PhD in 2014, following which I received comprehensive training at FDA-NCTR and University of New Mexico. During my career, I conducted original research in the area of cancer epigenomics and transcriptomics. While performing basic research in cancer biology and epigenetics, I became passionate about scientific discoveries in the area of bioinformatics and computational biology. In the Morin lab, I am applying a variety of bioinformatics tools to uncover genomic features of Burkitt lymphoma, a form of aggressive B-cell non-Hodgkin lymphoma. Specifically, my research is focused on comparing the genomic and molecular characteristics of adult and pediatric BL, Epstein-Barr virus (EBV) positive and negative BL, as well as endemic and sporadic cases, which will allow better diagnosis approaches and will ultimately facilitate development of more effective treatment strategies.
</p>
    
    
    
    
    <a href="https://github.com/kdreval"><img src="/images/team/github.jpg" width="52" height="52" /></a>
    
    
</li>

            
        
    
        
    
        
            
                <li>
    <h2><span class="member-name">Ryan Riley</span>, Bioinformatician</h2>
    
        <img class="float-left member-photo" src="/images/team/IMG_1330.jpg" />
    
    <p>I obtained my MSc in the Gregg Morin lab co-supervised by Ryan Morin and now I am working in Ryan Morin's lab co-supervised by Gregg Morin. My lab "claim to fame" is that my hair length exceeds the combined hair length of all the other Ryans and Morins. My interests include long walks on windy beaches at sunset, mantle cell lymphoma and proteomics.
</p>
    
    
    
    
    
</li>

            
        
    
        
            
                <li>
    <h2><span class="member-name">Dr. Elie Ritch</span>, Bioinformatician</h2>
    
        <img class="float-left member-photo" src="/images/team/Elie_crop.jpeg" />
    
    <p>I started my journey into bioinformatics and genomics in the Morin lab and here I am again after completing my PhD. It's just that fun!
</p>
    
    
    
    
    
</li>

            
        
    
        
            
                <li>
    <h2><span class="member-name">Carl-Adam Mattsson</span>, Research Programmer</h2>
    
        <img class="float-left member-photo" src="/images/team/adam.jpg" />
    
    <p>With an MSc. in Bioinformatics (UBC) I work as a Research Programmer, focusing on improving existing bioinformatic and computational infrastructure as well as the development and design of novel bioinformatic pipelines connected to Mature B-cell Lymphoma research facilitated by the Morin lab. With a background in cytogenetics and medical genetics, I am very interested in linking rare genetic variation to cancer predisposition and allowing for improved and effective treatment decisions in cancer.
</p>
    
    
    
    
    <a href="https://github.com/mattssca"><img src="/images/team/github.jpg" width="52" height="52" /></a>
    
    
</li>

            
        
    
        
            
                <li>
    <h2><span class="member-name">Jasper Wong</span>, Bioinformatician</h2>
    
        <img class="float-left member-photo" src="/images/team/jasper.JPG" />
    
    <p>Rare aggressive extra-nodal diffuse large B-cell lymphoma (DLBCL) remains unresponsive to standard treatments and has a poor prognosis. My projects aim to genetically characterize these extra-nodal lymphomas to better understand potential mechanisms driving their pathologies. One project involves investigating primary extranodal DLBCLs (specifically primary DLBCL in the testes and CNS), and understanding biological distinctions that separate these extranodal DLBCLs compared to their nodal counterparts, and to examine factors that promote their progression. The other project involves examining plasmablastic lymphoma, a disease typically involving immunocompromised individuals. Because of the rarity of this disease, very few studies have been able to examine the genomic landscape of this disease in depth. Coming from an epigenetics background, I have a particular interest in understanding the dynamics between the genome and epigenome in cancer.
</p>
    
        <a href="https://orcid.org/0000-0002-8140-6517"><img src="/images/team/orcid.png" width="50" height="50" /></a>
    
    
    
    
    <a href="https://github.com/Jwong684"><img src="/images/team/github.jpg" width="52" height="52" /></a>
    
    
</li>

            
        
    
        
            
                <li>
    <h2><span class="member-name">Sierra Gillis</span>, Research Assistant</h2>
    
        <img class="float-left member-photo" src="/images/team/sierra.jpg" />
    
    <p>I'm part of the Morin Lab to provide bioinfomatic support. I enjoy building pipelines for variant analyses, getting curious about Next-Generation Sequencing and alignment, and applying new techniques to investigate ctDNA. I hope to further my collaborative programming skills while doing so.
</p>
    
    
    
    
    <a href="https://github.com/gillissierra"><img src="/images/team/github.jpg" width="52" height="52" /></a>
    
    
</li>

            
        
    
        
            
                <li>
    <h2><span class="member-name">Manuela Cruz</span>, MSc student</h2>
    
        <img class="float-left member-photo" src="/images/team/cruz.jpg" />
    
    <p>I perform various computational analyses to assist in the classification of driver genes within Non-Hodgkin’s Lymphoma. By using computational methods designed to recognize recurrently mutated elements, it is possible to identify mutational hotspots within genes of interest. Hotspots, or genomic regions that contain a higher density of mutations than would be expected, can be recognized by analyzing the linear genomic sequence or the 3D structure of the expressed protein. 3D structure-based methods can supplement driver gene discovery by identifying regions in the tertiary structure that may mediate protein function and cell survival. I employ both types of methods for hotspot identification and driver gene discovery.
</p>
    
    
    
    
    
</li>

            
        
    
        
    
        
    
        
    
        
    
        
            
                <li>
    <h2><span class="member-name">Quratulain Qureshi</span>, MSc student (Co-supervised by Dr. Tim Audas)</h2>
    
        <img class="float-left member-photo" src="/images/team/annie.jpg" />
    
    <p>My research project aims to investigate the mechanisms of regulation of a gene encoding an RNA binding protein, HNRNPH1. HNRNPH1 was found to be recurrently mutated in Mantle Cell Lymphoma (MCL) patients, identified through whole genome sequencing in the Morin lab. Specifically, I am implementing in-vitro assays to detect secondary structures in the HNRNPH1 RNA, hypothesized to play important roles in its regulation. I am also testing whether the introduction of patient derived mutations disrupt these structures and contribute to MCL pathobiology.
</p>
    
    
    
    
    
</li>

            
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
        
    
    <!-- Non-current (alumni) -->
    
    
        <h1 class="post-title">Alumni</h1>
        
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
                
                    <li>
    <h2><span class="member-name">Kristena Daley</span>, MSc Student</h2>
    
        <img class="float-left member-photo" src="/images/team/kristena.jpg" />
    
    <p>I received my BSc in biology in 2019, with my honours thesis focusing on the assembly and annotation of the Atlantic deep-sea scallop transcriptome in order to determine differential gene expression in populations across Atlantic Canada. In the Morin lab, I am currently researching circulating tumor DNA (ctDNA) in various Non-Hodgkin’s lymphoma subtypes to identify mutation- and progression-based trends. My project aims to infer genome-wide methylation of Burkitt lymphoma (BL) and diffuse large B-cell lymphoma (DLBCL), as well as explore the benefits and drawbacks of common ctDNA quantification strategies for both BL and DLBCL patients across Canada.
</p>
    
    
    
    
    
</li>

                
            
        
            
        
            
        
            
        
            
        
            
        
            
        
            
                
                    <li>
    <h2><span class="member-name">Prince Kumar Lat</span>, Volunteer</h2>
    
        <img class="float-left member-photo" src="/images/team/prince.jpg" />
    
    <p>I completed my B.Tech in Bioengineering from IIT Kanpur, India in 2015. I did my B.Tech project at two different labs. First, at Max-Planck Institute, Germany I developed molecular biology techniques to study circular RNA. Second, at IIT Kanpur, where I applied computational tools to study olfactory receptor mechanism in Drosophila. I completed my PhD at Simon Fraser University in 2021, studying in-depth chemistry and biophysics of nucleic acids, including DNA/RNA catalysis and DNA nanotechnology. Towards the end of my PhD, I developed keen interest in applying machine learning models to solve biological problems. In the Morin lab, I aim to deploy robust classification and clustering models to enhance the current understanding on poorly classified B-cell lymphomas.
</p>
    
    
    
    
    <a href="https://github.com/princeklat03"><img src="/images/team/github.jpg" width="52" height="52" /></a>
    
    
</li>

                
            
        
            
                
                    <li>
    <h2><span class="member-name">Dr. Christopher Rushton</span>, PhD Student</h2>
    
        <img class="float-left member-photo" src="/images/team/crushton.jpg" />
    
    <p>My research project focuses on identifying mutations in patients with relapsed Diffuse Large B-Cell Lymphoma; specifically, the analysis of liquid biopsies obtained from these patients. As cells undergo apoptosis, apoptotic bodies containing DNA are released into the bloodstream, and this DNA is called cell-free DNA. In patients with cancer, a portion of this cell-free DNA originates from malignant cells, termed circulating-tumour DNA. Thus, we can obtain tumour genetic material simply by obtaining a blood sample (called a liquid biopsy), which is less invasive and significantly cheaper than traditional biopsies. Due to the unique nature of liquid biopsies, I am currently developing several pieces of software for analyzing and benchmarking existing mutation-calling tools for use with these types of samples. I am applying these tools to samples collected from several clinical trials where liquid biopsies are collected after patients are treated. By comparing the mutation landscape of these patients prior to and following treatment, we identify mutations enriched following treatment, which may contribute to tumour treatment resistance.
</p>
    
    
    
    
    <a href="https://github.com/ckrushton"><img src="/images/team/github.jpg" width="52" height="52" /></a>
    
    
</li>

                
            
        
            
                
                    <li>
    <h2><span class="member-name">Jeffrey Tang</span>, MSc Student</h2>
    
        <img class="float-left member-photo" src="/images/team/jtang.jpg" />
    
    <p>Recent studies have identified 4 or 5 genetic subgroups in DLBCL, which present a novel method of classification for alternative treatment strategies. My project aims to expand on classifying these subgroups in DLBCL using a machine learning approaches utilizing molecular indicators such as mutation status from a set of genes and hot spots, copy number status and cell-of-origin. I hope to consolidate subtypes from these studies as well as to capture novel groups to better describe clinical heterogeneity in DLBCL.
</p>
    
    
    
    
    <a href="https://github.com/jeffstang"><img src="/images/team/github.jpg" width="52" height="52" /></a>
    
    
</li>

                
            
        
            
                
                    <li>
    <h2><span class="member-name">Leah Tooman</span>, Research Assistant</h2>
    
        <img class="float-left member-photo" src="/images/team/ltooman.jpg" />
    
    <p>My role in the Morin Lab is to support the research that is being conducted by our students and postdoctoral fellows through sequencing and cell culture work. I am also responsible for monitoring and maintaining supplies and equipment, conducting research, and ensuring the lab is maintaining high ethical standards . I have been working in a similar position in different genetics based labs in Canada, New Zealand and the United States for over a decade. As passionate as I am about the field of science, I wish it was equally accessible to people of all genders and ethnicities, and from all socioeconomic backgrounds.
</p>
    
    
    
    
    
</li>

                
            
        
            
        
            
                
                    <li>
    <h2><span class="member-name">Dr. Bruno Grande</span>, Postdoctoral fellow, former PhD student</h2>
    
        <img class="float-left member-photo" src="/images/team/bgrande.jpg" />
    
    <p>As the first bioinformatics graduate student in the lab, I've played a role in several projects, each involving cancer genomics to some degree. Currently, my primary project is the genetic and molecular characterization of paediatric Burkitt lymphoma (BL). Defined by an epidemiological association with the Epstein–Barr virus and malaria, the endemic variant of BL has been understudied compared to sporadic BL, which is diagnosed in developed countries. I lead the analysis of whole genome and transcriptome sequencing data derived from over 100 cases, which has led to <a href="https://www.ncbi.nlm.nih.gov/pubmed/30617194">breakthroughs</a> in our understanding of BL pathogenesis. On the side, I'm an advocate for open and reproducible science, a Software Carpentry instructor teaching programming to researchers, and the co-founder of the Scientific Programming Study Group at SFU (SciProg.ca) and the annual Hackseq genomics hackathon.
</p>
    
        <a href="https://orcid.org/0000-0002-4621-1589"><img src="/images/team/orcid.png" width="50" height="50" /></a>
    
    
    <a href="https://scholar.google.ca/citations?user=xHoNHKEAAAAJ&amp;hl=en"><img src="/images/team/scholar.png" width="60" height="60" /></a>
    
    
    
    <a href="https://github.com/scientificbruno"><img src="/images/team/github.jpg" width="52" height="52" /></a>
    
    
    <a href="https://twitter.com/scientificbruno"><img src="/images/team/twitter.png" width="55" height="55" /></a>
    
</li>

                
            
        
            
                
                    <li>
    <h2><span class="member-name">Dr. Sarah Arthur</span>, Postdoctoral Fellow (former PhD student)</h2>
    
        <img class="float-left member-photo" src="/images/team/sarthur.jpg" />
    
    <p>My project in the lab is focused on the characterization of novel <a href="https://www.ncbi.nlm.nih.gov/pubmed/30275490">NFKBIZ 3’ UTR mutations</a> we previously discovered in DLBCL. These mutations were discovered through a project searching for genome-wide non-coding mutations and found to be significantly enriched in the ABC sub-type of DLBCL. My research involves functional analysis of these mutations in DLBCL cell lines and in vivo mouse xenograft models to determine how these mutations lead to activation of the NF-kB pathway and contribute to DLBCL. I am also working to determine the structure of the NFKBIZ 3’ UTR and how mutations lead to changes that affect the regulation of this gene.
</p>
    
    
    
    
    
</li>

                
            
        
            
                
                    <li>
    <h2><span class="member-name">Dr. Miguel Alcaide</span>, Research Associate</h2>
    
        <img class="float-left member-photo" src="/images/team/malcaide.jpg" />
    
    <p>My main role at the Morin lab focuses on the development and implementation of assays with potential utility in the genetic stratification, non-invasive monitoring and management of cancer patients. We have developed assays relying on <a href="https://www.ncbi.nlm.nih.gov/pubmed/28874686">targeted hybridization capture followed by high throughput sequencing</a> as well as assays relying on droplet digital PCR (ddPCR). I am particularly interested in the development and optimization of molecular techinques for the ultrasensitive and straightforward detection of tumour-derived DNA fragments. Detection and quantification of circulating tumour DNA (ctDNA) via duplex sequencing with semi-degenerate barcoded adapters and the use of wild-type specific hydrolysis probes <a href="https://www.ncbi.nlm.nih.gov/pubmed/27440511">(“inverted” ddPCR)</a> to detect multiple recurrent somatic mutations of clinical relevance are amongst my main contributions into this growing area of research. Currently, I am involved in different projects related to the genetic characterization of different types of Non-Hodgkin Lymphoma, and the detection, quantification and serial monitoring of ctDNA in cancer patients, including those enrolled in the personalized oncogenomics project (POG). We are also starting to apply our error-correction duplex sequencing approach for the sequencing of ancient DNA, including archaeological specimens of both pacific salmons and woolly dogs.
</p>
    
    
    <a href="https://scholar.google.ca/citations?hl=en&amp;user=5Jig5G4AAAAJ"><img src="/images/team/scholar.png" width="60" height="60" /></a>
    
    
    <a href="https://www.researchgate.net/profile/Miguel_Alcaide"><img src="/images/team/researchgate.png" width="60" height="60" /></a>
    
    
    
</li>

                
            
        
            
                
                    <li>
    <h2><span class="member-name">Aixiang Jiang</span>, Biostatistician</h2>
    
        <img class="float-left member-photo" src="/images/team/ajiang.jpg" />
    
    <p>I am a statistician and a bioinformatician at times. Although I had several years' experience in a molecular biology lab and did tissue/cell culture, DNA and RNA isolation, real-time PCR, quantitative and qualitative reverse transcriptase PCR (RT-PCR), cloning, western blots, southern blots, yeast 2-hybrid (Y2H) assay, microarray, even manually sequencing and so on; I have been focusing on statistical analysis since 2002.
My daily work involves data analysis, statistical consulting, and of course statistical programming. My research interesting, however, is to develop customized statistical algorithms to solve biological researches problems, especially for high dimensional data. My recent researches are to identify mutation peaks/ranges and build up new <a href="https://www.ncbi.nlm.nih.gov/pubmed/30523716">classification models</a> that can be transformed to different data sets and even different platforms.
</p>
    
    
    
    
    <a href="https://github.com/ajiangsfu"><img src="/images/team/github.jpg" width="52" height="52" /></a>
    
    
</li>

                
            
        
            
                
                    <li>
    <h2><span class="member-name">Shaghayegh Soudi</span>, Postdoctoral Fellow</h2>
    
        <img class="float-left member-photo" src="/images/team/photo_shaghayegh.png" />
    
    <p>I earned my PhD in Evolutionary Biology from Germany. Before joining the Morin lab in January 2021, I was a postdoc at university of California-Davis and UBC-University of Calgary working on understanding the genomic bases of adaptation and convergent evolution. Cancers evolve by a reiterative process of clonal expansion, genetic diversification and clonal selection within the adaptive landscapes of tissue ecosystems. The dynamics are complex, with highly variable patterns of genetic diversity and resulting clonal architecture which are closely related to cancer progression, resistance to therapy, and recurrences,therefore, our better understanding of the evolutionary landscape in cancer can help to develop advanced treatments matching appropriately selected molecular targets. My main interest is to apply evolutionary genomics tools to understand intra-tumour heterogeneity across multiple non-Hodgkin Lymphomas and to reconstruct their evolutionary history as part of the Pan-cancer analysis of B-cell lymphomas using whole-genome sequencing data. These analyses will eventually allow us to reveal the timing of specific genomic aberrations and changing mutational process over the course of cancer evolution.
</p>
    
    
    
    
    
</li>

                
            
        
            
                
                    <li>
    <h2><span class="member-name">Krishanna Campbell</span>, Undergraduate researcher</h2>
    
        <img class="float-left member-photo" src="/images/team/peanut.png" />
    
    <p>I am an undergraduate student in MBB at SFU and a long-term volunteer with the lab. My duties include the routine processing of blood samples to isolate plasma and cell pellet for later analysis ( eg. Circulating tumor DNA) along with purification of the cell-free DNA from plasmas utilizing the MagMax cell-free DNA isolation kit and subsequent procedures. I also assist in the canine B-cell lymphoma project specifically in the wet lab setting. I have a complete obsession with anything to do with animals and currently volunteer at a veterinary hospital. I plan to study veterinary medicine after completing my BSc.
</p>
    
    
    
    
    
</li>

                
            
        
            
                
                    <li>
    <h2><span class="member-name">Matt Cheung</span>, Undergraduate researcher</h2>
    
        <img class="float-left member-photo" src="/images/team/mcheung.jpg" />
    
    <p></p>
    
    
    
    
    
</li>

                
            
        
            
                
                    <li>
    <h2><span class="member-name">Matthew Nguyen</span>, Undergraduate bioinformatician (NSERC USRA)</h2>
    
        <img class="float-left member-photo" src="/images/team/mnguyen.jpg" />
    
    <p>My project focuses on using machine learning to detect and filter sequencing artifacts in formalin-fixed paraffin embedded (FFPE) tissues. This will enable more robust variant calling in FFPE tumours which remains a major limitation in the transition from fresh frozen tissues. I am also using interpretable machine learning approaches to classify subtypes of Non-Hodgkin's Lymphomas.
</p>
    
    
    
    
    <a href="https://github.com/matnguyen"><img src="/images/team/github.jpg" width="52" height="52" /></a>
    
    
</li>

                
            
        
            
                
                    <li>
    <h2><span class="member-name">Jack Hillman</span>, Undergraduate researcher (NSERC USRA)</h2>
    
        <img class="float-left member-photo" src="/images/team/jhillman.jpg" />
    
    <p>I assist with a variety of ongoing projects in the lab. I am currently performing hybridization capture to sequence several lymphoma-related genes in human and canine tumour samples.
</p>
    
    
    
    
    
</li>

                
            
        
            
                
                    <li>
    <h2><span class="member-name">Tony Ngo</span>, Undergraduate researcher</h2>
    
        <img class="float-left member-photo" src="/images/team/tngo.jpg" />
    
    <p>I am an undergraduate student completing a BSc in MBB and joined the laboratory after completing MBB 342. My honors project in the Morin lab focuses on the Fc-gamma receptor locus and the relationship between common germline copy number alterations and non-Hodgkin lymphoma. I am using custom multiplex droplet digital PCR assays to determine the copy number state of five genes in this locus across large cohorts of cases and controls.
</p>
    
    
    
    
    
</li>

                
            
        
            
                
                    <li>
    <h2><span class="member-name">Stephen Yu</span>, MSc Student</h2>
    
        <img class="float-left member-photo" src="/images/team/syu.jpg" />
    
    <p>Circulating tumour DNA (ctDNA) is rapidly becoming adopted as a sensitive biomarker for non-invasive monitoring of treatment response in many cancer types. Stephen's work helped to establish and validate numerous assays for quantifying ctDNA in plasma from lymphoma patients based on common hot spot mutations in genes such as EZH2 and STAT6. These assays demonstrated prognostic utility of ctDNA in multiple <a href="https://www.ncbi.nlm.nih.gov/pubmed/27166360">clinical trials</a> and helped describe <a href="http://clincancerres.aacrjournals.org/content/22/9/2290">patterns of clonal evolution in DLBCL</a>.
</p>
    
    
    
    
    
</li>

                
            
        
            
                
                    <li>
    <h2><span class="member-name">Kevin Bushell</span>, MSc Student</h2>
    
        <img class="float-left member-photo" src="/images/team/kbushell.jpg" />
    
    <p>Lymphoma is the most common cancer affecting pet dogs. Kevin used genomic techniques to identify mutations involved in lymphomagenesis in canine B-cell lymphoma. His work identified <a href="https://www.ncbi.nlm.nih.gov/pubmed/25468570">TRAF3</a> as one of the most commonly mutated genes in this cancer. Kevin also studied the potential utility of ctDNA in a variety of human cancers using several techniques including the OnTarget system from Boreal Genomics. He joined the laboratory full-time after graduating and has since transitioned to law school.
</p>
    
    
    <a href="https://scholar.google.ca/citations?user=OxP8_oIAAAAJ&amp;hl=en"><img src="/images/team/scholar.png" width="60" height="60" /></a>
    
    
    
    
</li>

                
            
        
            
                
                    <li>
    <h2><span class="member-name">Arezoo Mohajeri</span>, MSc Student</h2>
    
        <img class="float-left member-photo" src="/images/team/amohajeri.jpg" />
    
    <p>Arezoo generated and analyzed exome sequence data from diffuse large B-cell lymphoma (DLBCL) and mantle cell lymphoma (MCL) tumours. Her thesis project involved a combination of wet-lab work and bioinformatics including a meta-analysis of published and in-house exome data to identify new recurrently-mutated genes in MCL. Her discoveries a novel silent mutation hot spot in MAP3K14. She also demonstrated patterns of clonal evolution in relapsed DLBCL.
</p>
    
    
    
    <a href="https://www.researchgate.net/profile/Arezoo_Mohajeri"><img src="/images/team/researchgate.png" width="60" height="60" /></a>
    
    
    
</li>

                
            
        
            
                
                    <li>
    <h2><span class="member-name">Romene Sablok</span>, Undergraduate researcher</h2>
    
        <img class="float-left member-photo" src="/images/team/harry.jpg" />
    
    <p></p>
    
    
    
    
    
</li>

                
            
        
            
                
                    <li>
    <h2><span class="member-name">Jordan Davidson</span>, Undergraduate researcher</h2>
    
        <img class="float-left member-photo" src="/images/team/jdavidson.jpg" />
    
    <p></p>
    
    
    
    
    
</li>

                
            
        
            
                
                    <li>
    <h2><span class="member-name">Elie Ritch</span>, Undergraduate bioinformatician</h2>
    
        <img class="float-left member-photo" src="/images/team/harry.jpg" />
    
    <p></p>
    
    
    
    
    
</li>

                
            
        
            
                
                    <li>
    <h2><span class="member-name">Sepideh Alamouti</span>, Postdoctoral fellow</h2>
    
        <img class="float-left member-photo" src="/images/team/harry.jpg" />
    
    <p></p>
    
    
    
    
    
</li>

                
            
        
            
                
                    <li>
    <h2><span class="member-name">Dan Fornika</span>, Research Assistant/Bioinformatician</h2>
    
        <img class="float-left member-photo" src="/images/team/dfornika.jpg" />
    
    <p></p>
    
    
    
    
    
    <a href="https://twitter.com/morinryan"><img src="/images/team/twitter.png" width="55" height="55" /></a>
    
</li>

                
            
        
            
                
                    <li>
    <h2><span class="member-name">Selin Jessa</span>, Undergraduate bioinformatician</h2>
    
        <img class="float-left member-photo" src="/images/team/harry.jpg" />
    
    <p></p>
    
    
    
    
    
</li>

                
            
        
            
                
                    <li>
    <h2><span class="member-name">Marco Albuquerque</span>, Undergraduate bioinformatician</h2>
    
        <img class="float-left member-photo" src="/images/team/marco.jpg" />
    
    <p></p>
    
    
    
    
    <a href="https://github.com/marcoalbuquerque-sfu"><img src="/images/team/github.jpg" width="52" height="52" /></a>
    
    
</li>

                
            
        
            
                
                    <li>
    <h2><span class="member-name">Kristina Aluzaite</span>, Visiting Graduate Student</h2>
    
        <img class="float-left member-photo" src="/images/team/harry.jpg" />
    
    <p></p>
    
    
    
    
    
</li>

                
            
        
            
                
                    <li>
    <h2><span class="member-name">Stuart Zong</span>, Research Assistant</h2>
    
        <img class="float-left member-photo" src="/images/team/harry.jpg" />
    
    <p></p>
    
    
    
    
    
</li>

                
            
        
    
    </ul>
</div>

  </article>

</div>

      </div>
    </div>

    <footer class="site-footer">

  <div class="wrapper">

    <div class="footer-col-wrapper">
      <div class="footer-col footer-col-1">
        <p>The Morin Laboratory performs cutting-edge lymphoma research using a combination of dry lab and wet lab experiments.
</p>
      </div>

      <div class="footer-col footer-col-2">
        <ul class="social-media-list">
          
          <li>
            <a href="https://twitter.com/morinryan"><span class="icon icon--twitter"><svg viewBox="0 0 16 16"><path fill="#828282" d="M15.969,3.058c-0.586,0.26-1.217,0.436-1.878,0.515c0.675-0.405,1.194-1.045,1.438-1.809c-0.632,0.375-1.332,0.647-2.076,0.793c-0.596-0.636-1.446-1.033-2.387-1.033c-1.806,0-3.27,1.464-3.27,3.27 c0,0.256,0.029,0.506,0.085,0.745C5.163,5.404,2.753,4.102,1.14,2.124C0.859,2.607,0.698,3.168,0.698,3.767 c0,1.134,0.577,2.135,1.455,2.722C1.616,6.472,1.112,6.325,0.671,6.08c0,0.014,0,0.027,0,0.041c0,1.584,1.127,2.906,2.623,3.206 C3.02,9.402,2.731,9.442,2.433,9.442c-0.211,0-0.416-0.021-0.615-0.059c0.416,1.299,1.624,2.245,3.055,2.271 c-1.119,0.877-2.529,1.4-4.061,1.4c-0.264,0-0.524-0.015-0.78-0.046c1.447,0.928,3.166,1.469,5.013,1.469 c6.015,0,9.304-4.983,9.304-9.304c0-0.142-0.003-0.283-0.009-0.423C14.976,4.29,15.531,3.714,15.969,3.058z"/></svg>
</span><span class="username">morinryan</span></a>

          </li>
          

          
          <li>
            <a href="https://github.com/morinlab"><span class="icon icon--github"><svg viewBox="0 0 16 16"><path fill="#828282" d="M7.999,0.431c-4.285,0-7.76,3.474-7.76,7.761 c0,3.428,2.223,6.337,5.307,7.363c0.388,0.071,0.53-0.168,0.53-0.374c0-0.184-0.007-0.672-0.01-1.32 c-2.159,0.469-2.614-1.04-2.614-1.04c-0.353-0.896-0.862-1.135-0.862-1.135c-0.705-0.481,0.053-0.472,0.053-0.472 c0.779,0.055,1.189,0.8,1.189,0.8c0.692,1.186,1.816,0.843,2.258,0.645c0.071-0.502,0.271-0.843,0.493-1.037 C4.86,11.425,3.049,10.76,3.049,7.786c0-0.847,0.302-1.54,0.799-2.082C3.768,5.507,3.501,4.718,3.924,3.65 c0,0,0.652-0.209,2.134,0.796C6.677,4.273,7.34,4.187,8,4.184c0.659,0.003,1.323,0.089,1.943,0.261 c1.482-1.004,2.132-0.796,2.132-0.796c0.423,1.068,0.157,1.857,0.077,2.054c0.497,0.542,0.798,1.235,0.798,2.082 c0,2.981-1.814,3.637-3.543,3.829c0.279,0.24,0.527,0.713,0.527,1.437c0,1.037-0.01,1.874-0.01,2.129 c0,0.208,0.14,0.449,0.534,0.373c3.081-1.028,5.302-3.935,5.302-7.362C15.76,3.906,12.285,0.431,7.999,0.431z"/></svg>
</span><span class="username">morinlab</span></a>

          </li>
          
        </ul>
      </div>
    </div>

  </div>

</footer>


  </body>

</html>
